components/AccountsFooter.vue

```diff
@@ -1,54 +1,72 @@
 <template>
   <div class="accounts-footer-container">
-    <div class="container pr-10">
-      <div class="mr-10 pr-10">
-        <v-divider class="h-divider"></v-divider>
+    <v-container class="accounts-footer">
```

Comment:
A bunch of layout and style cleanup here (and in other components).

Note that I moved the dividers to the parent component (index.vue) to that they can exist in the correct containers and have the correct margins) (think "box in a box" model for layout and margins).